Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine3.17
+FROM python:3.12-alpine3.17
```

Comment:
For @felliott: now we use the python 3.12 image which installs node 18 by default

Reply:
huzzah!